mon-pix/translations/fr.json

```
@@ -262,6 +262,7 @@
             "timed-out-title": "Temps écoulé - Épreuve {stepNumber} sur {totalChallengeNumber}",
             "actions": {
                 "continue": "Poursuivre",
+                "go-to-next": "et je passe à la prochaine question",
```

Ça fait deux fois je passe :( 

```suggestion
                "go-to-next": "et je réponds à la question suivante",
```

---

Ah oui, effectivement, pas très jolie :s
Peut être "Je passe et je vais à la prochaine question"

---

Ou plus simplement :
```suggestion
                "go-to-next": "à la prochaine question",
```

je sais pas ce que ça donne dans le flow du reader, s'il fait une pause à la lectutre entre le texte du bouton et le `sr-only`